src/test/java/org/fundacionjala/fc/utils/JsonSchemaValidator.java

```diff
@@ -0,0 +1,26 @@
+package org.fundacionjala.fc.utils;
+
+import io.restassured.response.Response;
+import static io.restassured.module.jsv.JsonSchemaValidator.matchesJsonSchema;
+
+import java.io.File;
+
+public final class JsonSchemaValidator {
+
+    /**
+     * This avoids a default constructor for utility class.
+     */
+    private JsonSchemaValidator() {
+    }
+
+    /**
+     * A method that validate a http response from a given JsonSchema file.
+     * @param response the response obtained from the server.
+     * @param schemaFilePath the path of the JsonSchema file.
+     */
+    public static void validate(final Response response, final String schemaFilePath) {
+        File schemaFile = new File(schemaFilePath);
+        System.out.println(response.getBody());
```

Comment:
Please consider deleting this `System.out.println` statement. It's not necessary, we will log the request and responses in RequestManager.

Reply:
Thank you, I just delete that line at the last commit.